docs/changelogs/CHANGELOG-2.26.md

```diff
@@ -0,0 +1,320 @@
+- Add Support for Kubernetes 1.30 ([#13314](https://github.com/kubermatic/kubermatic/pull/13314))
+- Add support for Kubernetes 1.31 ([#13593](https://github.com/kubermatic/kubermatic/pull/13593))
+- Remove support for new Kubernetes 1.27 clusters. Existing clusters can still be reconciled, but must be upgraded before the KKP 2.28 ([#13710](https://github.com/kubermatic/kubermatic/pull/13710))
+
+#### Supported Versions
+
+- 1.28.9
+- 1.28.12
+- 1.28.13
+- 1.29.4
+- 1.30.3
+- 1.31.0
+
+### Cloud Providers
+
+#### Anexia
+
+- Update Anexia CCM to 1.5.6 ([#13501](https://github.com/kubermatic/kubermatic/pull/13501))
+
+#### AWS
+
+- Fix AWS nodes connectivity issue to the Metadata Service when using Cilium as the CNI (this impacted most visibly the EBS CSI driver not functioning correctly) ([#13554](https://github.com/kubermatic/kubermatic/pull/13554))
+- Update AWS CCM to v1.27.9, v1.28.9, v1.29.6, v1.30.3 ([#13495](https://github.com/kubermatic/kubermatic/pull/13495))
+
+#### Azure
+
+- Fix `csi` Addon not applying cleanly on Azure user clusters that were created with KKP <= 2.24 ([#13250](https://github.com/kubermatic/kubermatic/pull/13250))
+- Fix an issue with Azure support that prevented successful provisioning of user clusters on some Azure locations ([#13405](https://github.com/kubermatic/kubermatic/pull/13405))
+- Fix Azure CCM not being reconciled because of labelling changes ([#13334](https://github.com/kubermatic/kubermatic/pull/13334))
+- The azuredisk/azurefile CSI addons have been replaced with manifests based on the upstream Helm chart ([#13514](https://github.com/kubermatic/kubermatic/pull/13514))
+- Update Azure CCM / cloud node manager to 1.27.18, 1.28.10, 1.29.8, 1.30.4 ([#13496](https://github.com/kubermatic/kubermatic/pull/13496))
+- Change Azure load balancer SKU default value to Standard ([#13328](https://github.com/kubermatic/kubermatic/pull/13328))
+
+#### DigitalOcean
+
+- Update Digitalocean CCM to v0.1.54 ([#13497](https://github.com/kubermatic/kubermatic/pull/13497))
+
+#### GCP
+
+- Update GCP CCM to 30.0.0, 29.0.0 ([#13510](https://github.com/kubermatic/kubermatic/pull/13510))
+
+#### Hetzner
+
+- Update Hetzner CCM to 1.20.0 ([#13500](https://github.com/kubermatic/kubermatic/pull/13500))
+
+#### KubeVirt
+
+- Allow to use generic namespace name for KubeVirt in single namespace mode ([#13614](https://github.com/kubermatic/kubermatic/pull/13614))
+- Kubevirt provider waits for the etcdbackups to get deleted before removing the namespace, when a cluster is deleted ([#13635](https://github.com/kubermatic/kubermatic/pull/13635))
+- Allow the deployment of Kubevirt user clusters in the single namespace of the infrastructure cluster ([#13552](https://github.com/kubermatic/kubermatic/pull/13552))
+
+#### OpenStack
+
+- Explicitly configure OpenStack CCM with floating IP pool configured for user cluster instead of defaulting to first external network available ([#12975](https://github.com/kubermatic/kubermatic/pull/12975))
```

Comment:
We should consistently call it "OpenStack" (capital O, capital S) in all list items